QUARKUS-5661.md

```diff
@@ -0,0 +1,21 @@
+# QUARKUS-5661 Bump kubernetes-client-bom from 6.13.4 to 7.0.1
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5661
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/45259
+
+## Scope of the testing
+- Verify, that no regressions happen after the update 
```

Comment:
please specify type of regression you looked at ... build time, tests execution time, memory usage?

Reply:
None of the above. Our only test for these things (perflab) doesn't use any openshift extensions